client/.eslintignore

```diff
@@ -0,0 +1,2 @@
+dist/
+build/
```

Comment:
This should live in the root of repository. Or this file can be gone if you apply the next change.